elasticsearch-details/impersonation.md

```diff
@@ -138,6 +138,9 @@ For impersonation to work, some valid test settings should be created and saved.
 ![Impersonate tab](<../.gitbook/assets/impersonate_tab.png>)
 
 ## Add/edit auth mock user
+External services mock is used to simulate the response of existing authentication or authorization service like LDAP.
+You don't need to create a user account for configuration testing.
+You will only need to define users (and the groups which belong) returned by external services, listed in test settings.
```

Comment:
...(and their associated groups) that would normally be returned by the external services...